specs/subresourceintegrity/spec.markdown

```diff
@@ -610,15 +594,15 @@ failed resource with a different one.
 
 Whenever a user agent attempts to [obtain a resource][] pointed to by a
 `link` element that has a `rel` attribute with the value of `stylesheet`,
-insert the following step between steps 1 and 2:
+modify step 4 to read:
 
-1.  Set the [integrity metadata][] of the request to the value
-    of the element's `integrity` attribute.
-{:start="2"}
+Do a potentially CORS-enabled fetch of the resulting absolute URL, with the
+mode being the current state of the element's crossorigin content attribute,
+the origin being the origin of the link element's Document, the default origin
+behaviour set to taint, and the [integrity metadata][] of the request to the
+value of the element's `integrity` attribute.
 
-Additionally, in the paragraph specifying when to fire a `load` event, add
-"response with a `corrupt` integrity state" to the list of load failure
-reasons which are considered network errors.
+{:start="2"}
```

Comment:
TBH, I don't know what the {:start="2"} business does, and it doesn't modify the compiled index.html at all :-) Thus, I happily have changed it to "4".